pages/Addons.md

```diff
@@ -65,6 +65,7 @@ PotionExpansion | EpicPlayerA10 | <details><summary>expand</summary>An addon whi
 FlowerPower | NCBPFluffyBear | <details><summary>expand</summary>FlowerPower is a magic addon centered around experience and flowers. The base of each recipe requires various flowers, and different items are crafted along the way to make mid to endgame items. A majority of these items function by consuming experience in exchange for a certain effect or buff.</details> | [Readme](https://github.com/NCBPFluffyBear/FlowerPower/blob/master/README.md) | [GitHub](https://github.com/NCBPFluffyBear/FlowerPower/) | [Builds](https://thebusybiscuit.github.io/builds/NCBPFluffyBear/FlowerPower/master/)
 Galactifun | Seggan, Mooy1, GallowsDove, ProfElements | <details><summary>expand</summary>Galactifun is an addon inspired by ClayTech and the mod Galacticraft. It adds multiple planets, rockets, space stations, and more!</details> | [Readme](https://github.com/Slimefun-Addon-Community/Galactifun/blob/master/README.md) | [GitHub](https://github.com/Slimefun-Addon-Community/Galactifun/) | [Builds](https://thebusybiscuit.github.io/builds/Slimefun-Addon-Community/Galactifun/master/)
 FNAmplifications | FN_FAL113 | <details><summary>expand</summary>A four in one addon that includes my FN Solar Generator, Material Generator, Machinery and Power Xpansion Addons which are based from the addons made by the slimefun addon community.</details> | [Readme](https://github.com/FN-FAL113/FN-FAL-s-Amplifications/blob/main/README.md) | [GitHub](https://github.com/FN-FAL113/FN-FAL-s-Amplifications) | [Releases](https://github.com/FN-FAL113/FN-FAL-s-Amplifications/releases)
+ElementManipulation | lucasGithuber | <details><summary>expand</summary>A slimefun addon that adds elements and two more machines</details> | [Readme](https://github.com/lucasGithuber/Element-Manipulation/blob/master/README.md) | [GitHub](https://github.com/lucasGithuber/Element-Manipulation) | [Releases](https://github.com/lucasGithuber/Element-Manipulation/releases)
```

Comment:
> A slimefun addon that adds elements 

What kind of elements? Your description is quite vague at the moment

And if you ever plan on expanding this addon it might be better to stick with `more machines` over `two more machines`
```suggestion
ElementManipulation | lucasGithuber | <details><summary>expand</summary>A slimefun addon that adds elements and two more machines.</details> | [Readme](https://github.com/lucasGithuber/Element-Manipulation/blob/master/README.md) | [GitHub](https://github.com/lucasGithuber/Element-Manipulation) | [Releases](https://github.com/lucasGithuber/Element-Manipulation/releases)
```

Reply:
Ok